src/pages/Detail/Comments/InputForm.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const InputForm = ({ value, isValid, updateComment, handleValue }) => {
+  const ADD = {
+    type: 'add',
+    value: value,
+  };
```

Comment:
```suggestion
  const ADD_COMMENT = { type: 'add', value };
```
- 이정도는 한줄로 줄여써주는게 좋을거 같아요
- 그리고 동작을 나타내기 위한 객체라면 이름에서 해당 action을 표현하고 있으면 좋을거 같아요
- 근데 실제 사용되는 곳을 보니 한군데 모아 관리할게 아니라면 그냥 직접 써주는거 어떨까요? ADD라고 뚝 떨어져나와 있으니 조금 어색해보여서요

Reply:
감사합니다 종택님! 반영해서 수정하겠습니다!!